app.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const app = express();
+const { getTopics } = require('./controllers/topics-controllers')
+
+app.use(express.json());
+
+
+
+app.get('/api/topics', getTopics);
+
+app.use((req, res) => {
+    res.status(404).send({ msg: 'No Such Route'})
```

Comment:
Is this error actually firing? Custom error handlers should be using next so that the next error block will fire if this one isn't appropriate.

As Express handles missing routes I am unsure if this is actually doing anything.